packages/ui/src/views/chatflows/EmbedChat.jsx

```diff
@@ -173,6 +173,15 @@ const chatwindowConfig = (isReact = false) => {
                         // sendSoundLocation: "send_message.mp3", // If this is not used, the default sound effect will be played if sendSoundMessage is true.
                         receiveMessageSound: true,
                         // receiveSoundLocation: "receive_message.mp3", // If this is not used, the default sound effect will be played if receiveSoundMessage is true.
+                        inputHistory: {
```

Comment:
do we need a follow up PR on Embed (https://github.com/FlowiseAI/FlowiseChatEmbed) for this functionality?

Currently it only works for Flowise UI right?

Reply:
I was actually trying to add this to the FlowiseChatEmbed _first_ but I couldn't get it clear in my head how to point the Flowise project at that Embed module locally on my machine.  I have much of that work in flight already, and if we can chat through it for 10 minutes and I can share my screen, much of the ChatHistory code is on a branch on my local and the same.

But yes, right now it's only the chat widget within the Flowise Agent flow that has this.

Reply:
since we are hardcoding it in embed, can we remove this config?
 
![image](https://github.com/user-attachments/assets/b6ac13e8-5696-4d75-8f0e-ee26c6fcfda2)